### DISCUSSION TITLE
When using dzsave with --layout google is causing all of my tiles to be black and white.

### DISCUSSION THREAD

**Discussion by donnyv** [libvips/libvips#3747] [Q&A]
For some reason when generating tiles using these options. It generates all my tiles in black and white.
`vips dzsave A15B14.tif A15B14_TILES --layout google --suffix .png[compression=6,palette=TRUE] --tile-size 256`

I know they're in color because I checked them in QGIS.
Am I missing an option? The tiffs are geotifs.

QGIS properties
![image](https://github.com/libvips/libvips/assets/2224189/438a5a58-177a-40f1-baaa-fa87ace06316)

**Comment by jcupitt** [comment]
Hi @donnyv,

It works for me. Can you give more information?

- platform
- libvips version
- where the libvips binary came from
- sample TIF image that causes the problem
- sample incorrect output image you are getting

**Comment by donnyv** [comment]
platform: Windows 10
libvips version: 8.14.5
where the libvips binary came from: https://github.com/libvips/build-win64-mxe/releases/tag/v8.14.5
Folder with original geotifs https://drive.google.com/drive/folders/18DnDZ-XDojBDFq5QywayvrDLxR79lCcs?usp=sharing 

![2](https://github.com/libvips/libvips/assets/2224189/fa3f2a54-4131-48c2-bab4-a79741a489a6)
![0](https://github.com/libvips/libvips/assets/2224189/33d84642-57bf-4218-a3a7-34fd58404331)
![1](https://github.com/libvips/libvips/assets/2224189/fa713fa0-2664-4ad5-8ad3-42bade3dac88)

**Reply by donnyv** [reply]
I think its grabing only the first band when its creating the tiles.

From QGIS
![image](https://github.com/libvips/libvips/assets/2224189/a6be7c6c-1419-4a29-8a79-98bcde9de647)

**Comment by donnyv** [comment]
I got it to work. The original tif had a separate world file. I brought it into QGIS and saved it out as a geotif. But I picked Raw Data instead of Rendered image. That seemed to mess up the bands in the tif. 

![1](https://github.com/libvips/libvips/assets/2224189/787abd13-5e12-440a-9cdc-1386a500ab74)
![2](https://github.com/libvips/libvips/assets/2224189/f57d44a8-d88e-4759-9da9-6d2fb9745ddc)
![0](https://github.com/libvips/libvips/assets/2224189/1ab2e8e1-f338-44d5-bcea-b64355fd2bf9)
